apps/server/src/modules/account/mapper/account-response.mapper.ts

```diff
@@ -6,17 +6,17 @@ export class AccountResponseMapper {
 	static mapToResponseFromEntity(account: Account): AccountResponse {
 		return new AccountResponse({
 			id: account.id,
-			userId: account.userId.toString(),
-			activated: account.activated ?? false,
```

Comment:
why was the default removed?

Reply:
Because the underlying database model allows `undefined` and there are scenarios where an account is not fully initialized and `false` will not be the correct representation in that case.